ui/src/AppChooser.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {Suspense} from 'react';
+import LoadingOverlay from "./components/LoadingOverlay";
+
+const AppChooser = () => {
+
+    let MainComponent;
+    if (process.env.REACT_APP_OVERRIDE_IS_ENTERPRISE === "true") {
```

Comment:
should you also check for `window['isEnt']`?

Reply:
Nope - this split should be a replacement for this
but this is a good point because I can to remove this variable from the HTML file